python/pytorchserver/README.md

```diff
@@ -0,0 +1,146 @@
+Requirement already satisfied: pluggy!=0.10,<1.0,>=0.9 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->pytorchserver==0.1.0) (0.11.0)
+Requirement already satisfied: more-itertools>=4.0.0; python_version > "2.7" in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->pytorchserver==0.1.0) (7.0.0)
+Requirement already satisfied: wcwidth in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->pytorchserver==0.1.0) (0.1.7)
+Requirement already satisfied: atomicwrites>=1.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->pytorchserver==0.1.0) (1.3.0)
+Requirement already satisfied: attrs>=17.4.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->pytorchserver==0.1.0) (19.1.0)
+Requirement already satisfied: setuptools in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pytest->pytorchserver==0.1.0) (40.8.0)
+Requirement already satisfied: mypy-extensions<0.5.0,>=0.4.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from mypy->pytorchserver==0.1.0) (0.4.1)
+Requirement already satisfied: typed-ast<1.4.0,>=1.3.1 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from mypy->pytorchserver==0.1.0) (1.3.5)
+Installing collected packages: pytorchserver
+  Found existing installation: pytorchserver 0.1.0
+    Uninstalling pytorchserver-0.1.0:
+      Successfully uninstalled pytorchserver-0.1.0
+  Running setup.py develop for pytorchserver
+Successfully installed pytorchserver
+```
+
+To run tests, please change the test file to point to your model.pt file. Then run the following command:
+
+```bash
+make test
+```
+
+The following shows the type of output you should see:
+
+```
+pytest -W ignore
+=========================================================== test session starts ============================================================
+platform darwin -- Python 3.7.3, pytest-4.5.0, py-1.8.0, pluggy-0.11.0
+rootdir: /Users/animeshsingh/go/src/github.com/kubeflow/kfserving/python/pytorchserver
+plugins: tornasync-0.6.0.post1
+collected 1 item                                                                                                                           
+
+pytorchserver/test_model.py .                        
+```
+
+To run static type checks:
+
+```bash
+mypy --ignore-missing-imports pytorchserver
+```
+
+An empty result will indicate success.
+
+## Building your own PyTorch server Docker Image
+
+You can build and publish your own image for development needs. Please ensure that you modify the kfservice files for PyTorch in the api directory to point to your own image.
+
+To build your own image, run
+
+```bash
+docker build -t animeshsingh/pytorchserver -f pytorch.Dockerfile .
```

Comment:
docker_user_name

Reply:
Is there an env var thats typically set to this value?

Reply:
Chnaged to docker_user_name